firebase-appdistribution-api/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -179,6 +182,76 @@ public interface FirebaseAppDistribution {
    */
   void startFeedback(@NonNull CharSequence infoText, @Nullable Uri screenshot);
 
+  /**
+   * Displays a notification that, when tapped, will take a screenshot of the current activity, then
+   * start a new activity to collect and submit feedback from the tester along with the screenshot.
+   *
+   * <p>On Android 13 and above, this method requires the <a
+   * href="https://developer.android.com/develop/ui/views/notifications/notification-permission">runtime
+   * permission for sending notifications</a>: {@code POST_NOTIFICATIONS}. If your app targets
+   * Android 13 (API level 33) or above, you should <a
+   * href="https://developer.android.com/training/permissions/requesting">request the
+   * permission</a>.
+   *
+   * <p>When the notification is tapped:
+   *
+   * <ol>
+   *   <li>If the app is open, take a screenshot of the current activity
+   *   <li>If tester is not signed in, presents the tester with a Google Sign-in UI
+   *   <li>Starts a full screen activity for the tester to compose and submit the feedback
+   * </ol>
+   *
+   * <p>On Android 8 and above, the notification will be created in its own notification channel.
+   *
+   * @param infoTextResourceId string resource ID of text to display to the tester before collecting
+   *     feedback data (e.g. Terms and Conditions)
+   * @param importance the level of interruption for the feedback notification channel. Once the
+   *     channel's importance is set it cannot be changed except by the user. See {@link
+   *     NotificationChannel#setImportance}. On platforms below Android 8, the importance will be
+   *     translated into a comparable notification priority (see {@link
```

Comment:
If we are doing some translation, would it make sense to create an enum as opposed to relying on the platform constants being passed in?

Reply:
Here's what that would look like: https://github.com/firebase/firebase-android-sdk/pull/4205/files

Let me know if you think we should go through an API review for this.

Reply:
I like it (just left one comment for simplification). I don't think we need an API review now. But after EAP we should go through an API review (again) for everything. Many small details changed - and might keep changing - e.g. whether to accept `CharSequence` or `@StringRes int` or both.